.prettierrc

```diff
@@ -0,0 +1,14 @@
+{
+  "tabWidth": 2,
```

Comment:
This and the other whitespace settings should be deffered to an EditorConfig file instead. Both Prettier, IDEs, etc... will respect the values from that file